react/features/mobile/navigation/components/RootNavigationContainer.js

```diff
@@ -7,6 +7,8 @@ import { SafeAreaProvider } from 'react-native-safe-area-context';
 
 import { connect } from '../../../base/redux';
 import { DialInSummary } from '../../../invite';
+import LoadConfigOverlay
```

Comment:
Let's move this to features/mobile/navigation, and make a it a proper JitsiScreen no? Also, I think we should navigate to it more explicitly, instead of using the overlay system.

Reply:
JitsiScreen is a bit tricky in this case because it has a diff structure than OverlayFrame. Will come back to this.